libs/ngx-charts-on-fhir/src/lib/data-layer-list/shared-data-layer-list.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class SharedDataLayerListService {
+  subject$ = new BehaviorSubject<boolean>(false);
+
+  setOptionPanelValue(optionPanelValue: boolean) {
+    this.subject$.next(optionPanelValue);
+  }
+  getOptionPanelValue(): boolean {
+    return this.subject$.value;
+  }
```

Comment:
can we rename these to match the corresponding component property "showAdvancedOptions"?